bin/validate-helm

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"
+
+source ./test-workflow/utils.sh
+source ../helm/common/utils.sh
+
+helm_tests_return_code=0
+
+tests=(happy incorrect_url incorrect_cert incorrect_auth_id)
+
+print_usage() {
+  echo "Usage:"
+  echo "    This script will validate the helm chart tests"
+  echo ""
+  echo "Syntax:"
+  echo "    $0  [Options]"
+  echo "    Options:"
+  echo "    -h            Show help"
+  echo "    -c            Create the Kind cluster"
+  echo "    -d            Delete the Kind cluster"
+  echo "    -t            run one test"
+}
+
+
+function pushd() {
+    command pushd "$@" > /dev/null
+}
+
+function popd() {
+    command popd "$@" > /dev/null
+}
+
+function create_cluster() {
+# Create kind cluster
+    ./0_prep_conjur_in_kind.sh
+    . ./1_prep_env.sh
+    ./2_admin_load_conjur_policies.sh
+    ./3_admin_init_conjur_cert_authority.sh
+}
+
+function delete_cluster() {
+
+    kind delete cluster
+}
+
+readonly APPLIANCE_URL="https://conjur-oss.conjur-oss.svc.cluster.local"
+
+function git_clone() {
+    rm -rf conjur-authn-k8s-client
+    git clone https://github.com/cyberark/conjur-authn-k8s-client.git
+    pushd conjur-authn-k8s-client/helm/conjur-config-cluster-prep
+    ./bin/get-conjur-cert.sh -u $APPLIANCE_URL -i -v -s
+}
```

Comment:
Should there be a `popd` here in `git_clone()`? i.e. have the function restore state as much as it can when it has done its thing?